contracts/core/UserOperationLib.sol

```diff
@@ -65,12 +64,24 @@ library UserOperationLib {
         return abi.encode(
             sender, nonce,
             hashInitCode, hashCallData,
-            callGasLimit, verificationGasLimit, preVerificationGas,
+            accountGasLimits, preVerificationGas,
             maxFeePerGas, maxPriorityFeePerGas,
             hashPaymasterAndData
         );
     }
 
+    function unpackAccountGasLimits(
+        bytes32 accountGasLimits
+    ) internal pure returns(uint128 validationGasLimit, uint128 callGasLimit) {
+        return (uint128(bytes16(accountGasLimits)), uint128(uint256(accountGasLimits)));
+    }
+
+    function unpackPaymasterStaticFields(
```

Comment:
is it ever used?

Reply:
Maybe we should have these helper functions (and use them in our EntryPoint/samples) since the logic to unpack is not trivial?